<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthDialog</class>
 <widget class="QDialog" name="AuthDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>165</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>434</width>
    <height>165</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>434</width>
    <height>165</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Attention!</string>
  </property>
  <property name="styleSheet">
   <string>background-color: rgb(58,58,58); 
selection-background-color: rgb(67, 67, 67);
color: rgb(255, 255, 255);
</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>External device attempts to get control over your workstation. </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ip address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Device name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Device id:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ipAddress_">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ip goes here</string>
   </property>
  </widget>
  <widget class="QLabel" name="deviceName_">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>device name goes here</string>
   </property>
  </widget>
  <widget class="QLabel" name="deviceId_">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>device id goes here</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>440</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-image: url(:/Resources/SeparatorBottom.png); 
border: none;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>611</x>
     <y>141</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(46, 46, 46);
border:none;</string>
   </property>
   <widget class="QComboBox" name="endpoints_">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>3</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>QComboBox {
background-color: rgb(46,46,46);
border: none;
}</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>128</y>
     <width>440</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(46, 46, 46);
border:none;</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-image: url(:/Resources/ButtonNormal.png); 
border: none;
color: rgb(255, 255, 255);
font: bold 10pt &quot;Lucida Grande&quot;;</string>
    </property>
    <property name="text">
     <string>Allow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>346</x>
      <y>0</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-image: url(:/Resources/ButtonNormal.png); 
border: none;
color: rgb(255, 255, 255);
font: bold 10pt &quot;Lucida Grande&quot;;</string>
    </property>
    <property name="text">
     <string>Deny</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>0</y>
      <width>133</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>border-image: url(:/Resources/ButtonNormal.png) 0 5 0 5;
border-top: 0px transparent;
border-bottom: 0px transparent;
border-right: 5px transparent;
border-left: 5px transparent;
color: rgb(255, 255, 255);
font: bold 10pt &quot;Lucida Grande&quot;;</string>
    </property>
    <property name="text">
     <string>Allow &amp;&amp; Remember</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AuthDialog</receiver>
   <slot>allowed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AuthDialog</receiver>
   <slot>denied()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>AuthDialog</receiver>
   <slot>remembered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Allowed()</slot>
  <slot>Denied()</slot>
  <slot>remembered()</slot>
 </slots>
</ui>
